app/components/support_interface/tile_component.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="app-card<%= !regular_type? ? " app-card--#{type_colour}" : '' %>">
+  <span class="<%= count_class %>"><%= count %></span>
+  <span><%= label %></span>
```

Comment:
```suggestion
  <%= label %>
```

Unnecessary span? Can't find CSS that targets it.

Reply:
Thanks, I've removed this now.